app/git/utils.py

```diff
@@ -573,9 +573,12 @@ def get_interested_actions(github_url, username, email=''):
 
             for pr_action in all_pr_actions:
                 if 'actor' in pr_action:
-                    gh_user = pr_action['actor']['login']
+                    if (pr_action['actor']):
+                        gh_user = pr_action['actor']['login']
```

Comment:
What do you think of a oneliner here, like `gh_user = pr_action.get('actor', {}).get('login')`?

Reply:
I think its pretty slick personally, but I can see how it may be a bit unclear whats happening. What did you mean by "nomad"?